src/sql/util.py

```diff
@@ -363,3 +387,30 @@ def show_deprecation_warning():
         "raise an exception in the next major release so please remove it.",
         FutureWarning,
     )
+
+
+def is_saved_snippet_or_table_exists(table, schema=None, task=None):
+    """
+    Check if the referenced table is a snippet
+    Parameters
+    ----------
+    table : str,
+        name of table
+
+    schema : str, default None
+        Name of the schema
+
+    task: str, default None
+        Command calling this function
+
+    Returns
+    -------
+    with_ : str, default None
+        name of the snippet
+    """
+    with_ = None
+    if is_saved_snippet(table, task):
```

Comment:
I think `task` shouldn't be a parameter of `is_saved_snippet`. Can refactor this a bit:
```
if is_saved_snippet(table):
    print(f"{msg} using saved snippet : {table}")
```
And delete the print statement inside ` is_saved_snippet`

Reply:
Made this change, now we send a message in `is_saved_snippet_or_table_exists` itself.

Reply:
If it's clear anyways then I think we can remove it

Reply:
Sure! removing it